Sources/SimultaneouslyScrollView/Extensions/MulticastScrollViewDelegate.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit;
+
+class MulticastScrollViewDelegate: NSObject, UIScrollViewDelegate {
+    // Store weak references to delegates
+    private var delegates = NSHashTable<UIScrollViewDelegate>.weakObjects()
```

Comment:
We experienced in the past that the `NSHashTable.weakObjects` does not always work as expected when it comes to holding its objects `weak` 🤔 

I already had a custom implementation of this at some point which we could bring back: https://github.com/stonko1994/SimultaneouslyScrollView/pull/14/files#diff-7ec2bbf77a97574f56e797ad34ba0a592925aecac334b37ee9ad57db02470632

Reply:
Hm. What problems did you encounter? I'm ok with using your version though.